### PR TITLE
Add membership type field to the provider API

### PR DESCRIPTION
**Pull request by cttengsfmta**
Add membership type field to the provider API. We presume that more values could be added to the list of membership types.

### CODE REVIEW

File: provider/README.md [hunk 1]
```diff
@@ -168,6 +169,15 @@ Gets all trips within that bounding-box where any point inside the `route` is in
...
 
+### Membership Types
+
+| `membership_type` |
+|--------------|
+| subscriber             |
+| subscriber_low_income  |
+| single_ride            |
+| single_ride_low_income |
+
```
Comment (hunterowens) [inline provider/README.md]: @asadowns @noonhub @c-fang any missing types here?

**Comment by cttengsfmta** [comment]
Sorry about the bad push there. Hopefully nobody saw that.

**Comment by dyakovlev** [comment]
I'm missing some context here. What is this membership of? Can that be a subheading in that section or a description somewhere?

**Comment by hunterowens** [comment]
@cttengsfmta I'm gonna close this for now and you can reopen once you have a clean push. Thanks!

**Pull request closed without merging**